herbert_core/classes/data_loaders/@rundata/rundata.m

```diff
@@ -433,6 +433,17 @@
         function det=get.det_par(this)
             det = get_loader_field_(this,'det_par');
         end
+        function det=get_det_par_rows(this)
```

Comment:
I will agree with you on the general principle of your comment, but think it will be easier to do this as a post-construction exercise, after the code has settled. In the meantime, it is reassuring that these are all the fixes that need adding - I don't need any further swapping over the test suites. The basic message coming out is that detpar in sqw has been columnised - it's really the behaviour of IX_detector_bank now - and we prefer columns. However, detpar in rundata (still a detpar struct) is in rows. That's what we should fix post-construction.